test/functional/kubernetes/kubernetes_test.go

```diff
@@ -129,6 +130,25 @@ func Test_TutorialApplication_KubernetesManifests(t *testing.T) {
 
 		_, err = client.Get(ctx, deployment.Name, nil)
 		require.NoError(t, err)
+
+		// Get the list of pods
+		labelSelector := labels.Set{"app": "demo"}.AsSelector().String()
```

Comment:
Checking if crashloopbackoff has occured (since there could be other reasons for pods to be restarted). 
@rynowak , pls review if this is correct.